frontend/src/api/index.ts

```diff
@@ -23,6 +32,9 @@ export const getPlaylists = async (
     )}&offset=${encodeURIComponent(offset)}`,
     { credentials: "include" }
   );
+  if (response.status === 401) {
```

Comment:
Ooh nice, I haven't tested this branch but this looks like it would improve the initial UX definitely!